File: Server/src/main/java/com/api/server/foodb/Content.java
package com.api.server.foodb;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "contents")
public class Content {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "source_id")
    private Integer sourceId;

    @Size(max = 255)
    @Column(name = "source_type")
    private String sourceType;

    @NotNull
    @Column(name = "food_id", nullable = false)
    private Integer foodId;

    @Size(max = 255)
    @Column(name = "orig_food_id")
    private String origFoodId;

    @Size(max = 255)
    @Column(name = "orig_food_common_name")
    private String origFoodCommonName;

    @Size(max = 255)
    @Column(name = "orig_food_scientific_name")
    private String origFoodScientificName;

    @Size(max = 255)
    @Column(name = "orig_food_part")
    private String origFoodPart;

    @Size(max = 255)
    @Column(name = "orig_source_id")
    private String origSourceId;

    @Size(max = 255)
    @Column(name = "orig_source_name")
    private String origSourceName;

    @Column(name = "orig_content", precision = 15, scale = 9)
    private BigDecimal origContent;

    @Column(name = "orig_min", precision = 15, scale = 9)
    private BigDecimal origMin;

    @Column(name = "orig_max", precision = 15, scale = 9)
    private BigDecimal origMax;

    @Size(max = 255)
    @Column(name = "orig_unit")
    private String origUnit;

    @Lob
    @Column(name = "orig_citation")
    private String origCitation;

    @NotNull
    @Lob
    @Column(name = "citation", nullable = false)
    private String citation;

    @Size(max = 255)
    @NotNull
    @Column(name = "citation_type", nullable = false)
    private String citationType;

    @Column(name = "creator_id")
    private Integer creatorId;

    @Column(name = "updater_id")
    private Integer updaterId;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

    @Size(max = 255)
    @Column(name = "orig_method")
    private String origMethod;

    @Size(max = 255)
    @Column(name = "orig_unit_expression")
    private String origUnitExpression;

    @Column(name = "standard_content", precision = 15, scale = 9)
    private BigDecimal standardContent;

    @Size(max = 255)
    @Column(name = "preparation_type")
    private String preparationType;

    @Column(name = "export")
    private Byte export;

}

File: DotNetServer/DotNetServer/Entities/Content.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Entities;

[Table("contents")]
[Index("SourceId", "SourceType", "FoodId", Name = "content_source_and_food_index")]
public partial class Content
{
    [Key]
    [Column("id")]
    public int Id { get; set; }

    [Column("source_id")]
    public int? SourceId { get; set; }

    [Column("source_type")]
    public string? SourceType { get; set; }

    [Column("food_id")]
    public int FoodId { get; set; }

    [Column("orig_food_id")]
    [StringLength(255)]
    public string? OrigFoodId { get; set; }

    [Column("orig_food_common_name")]
    [StringLength(255)]
    public string? OrigFoodCommonName { get; set; }

    [Column("orig_food_scientific_name")]
    [StringLength(255)]
    public string? OrigFoodScientificName { get; set; }

    [Column("orig_food_part")]
    [StringLength(255)]
    public string? OrigFoodPart { get; set; }

    [Column("orig_source_id")]
    [StringLength(255)]
    public string? OrigSourceId { get; set; }

    [Column("orig_source_name")]
    [StringLength(255)]
    public string? OrigSourceName { get; set; }

    [Column("orig_content")]
    [Precision(15, 9)]
    public decimal? OrigContent { get; set; }

    [Column("orig_min")]
    [Precision(15, 9)]
    public decimal? OrigMin { get; set; }

    [Column("orig_max")]
    [Precision(15, 9)]
    public decimal? OrigMax { get; set; }

    [Column("orig_unit")]
    [StringLength(255)]
    public string? OrigUnit { get; set; }

    [Column("orig_citation", TypeName = "mediumtext")]
    public string? OrigCitation { get; set; }

    [Column("citation", TypeName = "mediumtext")]
    public string Citation { get; set; } = null!;

    [Column("citation_type")]
    [StringLength(255)]
    public string CitationType { get; set; } = null!;

    [Column("creator_id")]
    public int? CreatorId { get; set; }

    [Column("updater_id")]
    public int? UpdaterId { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }

    [Column("orig_method")]
    [StringLength(255)]
    public string? OrigMethod { get; set; }

    [Column("orig_unit_expression")]
    [StringLength(255)]
    public string? OrigUnitExpression { get; set; }

    [Column("standard_content")]
    [Precision(15, 9)]
    public decimal? StandardContent { get; set; }

    [Column("preparation_type")]
    [StringLength(255)]
    public string? PreparationType { get; set; }

    [Column("export")]
    public sbyte? Export { get; set; }
}
